.github/workflows/test.yml

```diff
@@ -53,7 +53,8 @@ jobs:
         terraform:
           - '0.12.30'
           - '0.13.6'
```

Comment:
```suggestion
          - '0.12.31'
          - '0.13.7'
```

Reply:
Not sure why GitHub opened multiple comments, but going to also include this for consistency. 👍